sdk/src/main/java/com/google/cloud/dataflow/sdk/runners/inprocess/InProcessPipelineRunner.java

```diff
@@ -174,21 +175,21 @@ public InProcessPipelineOptions getPipelineOptions() {
    */
   public static interface InProcessExecutor {
```

Comment:
Might make sense as top level package-private? I guess it depends how it is eventually used.